webapp/src/pages/Settings.tsx

```diff
@@ -548,8 +569,40 @@ const Settings: React.FC = () => {
     </FormGroup>
   );
 
+  const displayAnalysesCustomizationGeneralSection = () => (
+    <Box display="flex" justifyContent="flex-start" gap={5} alignItems="center">
+      <FormControl variant="standard" sx={{ m: 1, width: 120 }}>
+        <InputLabel id="supported-language-input-label">
+          supported_language
+        </InputLabel>
+        <Select
```

Comment:
For the `InputLabel`'s `id` to be useful, you must add it as the `labelId` of the `Select`.
```suggestion
        <InputLabel id="language-input-label">
          language
        </InputLabel>
        <Select
          labelId"language-input-label"
```

Reply:
Please review my suggestion carefully. The `InputLabel`'s `id` must be the same string as the `Select`'s `labelId`. That's what links them together for the browsers' accessibility features.

Reply:
Okay. My Apologise. I did not understand the values of both IDs correctly should be the same. I will remove the supported wherever it's inappropriate